src/Component.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
-
+import PropTypes from 'prop-types';
 
 const MAX_MOVE = 20;
 
 
 const extractCoordinates = ({changedTouches}) =>
   ({x: changedTouches[0].screenX, y: changedTouches[0].screenY});
 
-
-export const ReactPageClick = React.createClass({
+class ReactPageClick extends React.Component {
```

Comment:
Please, also switch to `React.PureComponent` so we can remove `import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';` line and shouldComponentUpdate too